R/oc_forward.R

```diff
@@ -0,0 +1,328 @@
+#' Forward geocoding
+#'
+#' Forward geocoding, from placename to latitude and longitude tuple(s).
+#'
+#' @param placename A character vector with the placename(s) to be geocoded.
+#'   Required. See
+#'   \href{https://github.com/OpenCageData/opencagedata-misc-docs/blob/master/query-formatting.md}{OpenCage's
```

Comment:
are we sure this is a document that won't disappear? nice one in any case.

Reply:
it won't disappear